README.md

```diff
@@ -892,7 +892,8 @@ var log = bunyan.createLogger({
         type: 'rotating-file',
         path: '/var/log/foo.log',
         period: '1d',   // daily rotation
-        count: 3        // keep 3 back copies
+        count: 3,        // keep 3 back copies,
+        maxSizeKB: 1024 // rotate every 1024KB
```

Comment:
You might also consider adding some instructions explaining how and where to set "logOnlyMinLevel" parameter.
I'm sorry if it's obvious and I didn't get it. :/